app/src/components/course/index.jsx

```diff
@@ -1,39 +1,20 @@
-import React, { useState, useEffect } from "react";
-import { getCourses } from "src/api/course";
-import CourseTile from "./CourseTile";
-
+import React, { useContext } from "react";
+import { users, UserContext } from "../user/UserContext";
+import StaffCourse from "./staff/StaffCourse";
 
 function Course() {
-  const [courses, setCourses] = useState([]);
-
-  useEffect(() => {
-    getAllJobs();
-
-    async function getAllJobs() {
-      const coursesReturnedFromBackend = await getCourses();
-      setCourses(coursesReturnedFromBackend);
-    }
-  }, []);
-
-
-  const renderCourses = courses.map((course, index) => (
-    <CourseTile
-      key={index}
-      courseId={course.courseId}
-      courseName={course.courseName}
-      courseDesc={course.courseDesc}
-      courseStatus={course.courseStatus}
-      registrationStatus={course.registrationStatus}
-      completionStatus={course.completionStatus}
-    />
-  ));
-
-  return (
-    <div className='flex flex-col container w-9/12 max-w-7xl mt-10 p-10 mx-auto w-full bg-white rounded-lg shadow-lg shadow-blue-200'>
-      <h1 className='text-3xl text-left font-bold'>Courses</h1>
-      {courses.length === 0 ? "No Courses Found" : renderCourses}
-    </div>
-  );
+  const [user, setUser] = useContext(UserContext);
+
+  switch (user || "login") {
+    case users.staff.name:
+      return <StaffCourse/>;
+    case users.hr.name:
+      return <div/>;
+    case users.manager.name:
+      return <div/>;
+    default:
+      return <div/>;
```

Comment:
Should we return default StaffCourse? We can just assume we'll not have intruders and it might help us reduce bugs. 

And should return<div></div>